ArcGISRuntimeSDKQt_CppSamples/AR/DisplayScenesInTabletopAR/android/AndroidManifest.xml

```diff
@@ -0,0 +1,94 @@
+<?xml version='1.0' encoding='utf-8'?>
+<manifest package="org.qtproject.example" xmlns:android="http://schemas.android.com/apk/res/android" android:versionName="-- %%INSERT_VERSION_NAME%% --" android:versionCode="-- %%INSERT_VERSION_CODE%% --" android:installLocation="auto">
+    <uses-sdk android:minSdkVersion="24" android:targetSdkVersion="28"/>
```

Comment:
Are these correct? These are quite high targets for Android.

Reply:
Based on my understanding of the readme for the toolkit, yes. 

> Set the minimum SDK version android:minSdkVersion to 24 for AR required app and 14 for AR optional app.

The above is found [here](https://github.com/Esri/arcgis-runtime-toolkit-qt/blob/master/Common/AR/README.md#configuration-for-android-devices).